includes/admin/class-wcmp-export-consignments.php

```diff
@@ -174,6 +174,10 @@ public function setCustomItems(): void
                 return;
             }
 
+            if ($product->is_virtual()) {
```

Comment:
Getver wat een grote loop 😢  klopt die return hierboven wel of had dat ook eigenlijk een continue moeten zijn? Weet dat het niet direct met je codes te maken heeft maar als je er toch bent... En straks komt het ineens uit de test omdat er grondiger naar gekeken gaat worden!